be/src/io/fs/azure_obj_storage_client.cpp

```diff
@@ -75,6 +78,40 @@ ObjectStorageResponse do_azure_client_call(Func f, const ObjectStoragePathOption
     return {};
 }
 
+struct AzureBatchDeleter {
```

Comment:
type alias the name space for readability, such as using asb = Azure::Storage::Blobs